### PR TITLE
Clarify example. Closes #1622

### LINKED ISSUE #1622: Object.prototype.isPrototypeOf
Replace existing example of the [isPrototypeOf] ([MDN]) by the following:

```js
function Foo() {}
function Bar() {}

Bar.prototype = Object.create(Foo.prototype);

const bar = new Bar();

console.log(Foo.prototype.isPrototypeOf(bar));
// expected output: true

console.log(Bar.prototype.isPrototypeOf(bar));
// expected output: true
```
Current example is vague.

[isPrototypeOf]: https://github.com/mdn/interactive-examples/blob/master/live-examples/js-examples/object/object-prototype-isprototypeof.js
[MDN]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/isPrototypeOf

### PR DESCRIPTION
**Pull request by hinell**
More details at #1622

### CODE REVIEW

File: live-examples/js-examples/object/object-prototype-isprototypeof.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+function Bar() {}
 
-object1.prototype = Object.create(object2.prototype);
+Bar.prototype = Object.create(Foo.prototype);
 
-const object3 = new object1();
+const bar = new Bar();
 
-console.log(object1.prototype.isPrototypeOf(object3));
-// expected output: true
```
Comment (velusgautam) [inline live-examples/js-examples/object/object-prototype-isprototypeof.js:9]: Why you deleted expected outputs?

File: live-examples/js-examples/object/object-prototype-isprototypeof.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+console.log(Foo.prototype.isPrototypeOf(bar));
```
Comment (wbamberg) [inline live-examples/js-examples/object/object-prototype-isprototypeof.js:8]: This should have:

```
 // expected output: true
```

after it

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks @hinell . I like this change but it should have an `// expected output` line for both `console.log()` statements.

**Review by wbamberg — APPROVED** [review]
Thanks @hinell !

**Pull request merged**